src/tmle_inputs/from_actors.jl

```diff
@@ -208,11 +233,18 @@ function tmle_inputs_from_actors(parsed_args)
 
     # Parameter files
     variables = TargeneCore.get_variables(pcs, traits, extraW, extraC, extraT)
-    TargeneCore.parameters_from_actors(
-        bqtls, transactors, data, variables, orders, outprefix; 
+
+    # Loop through each TF present in bqtls file 
+    tfs = unique(bqtls.TF)
+    for tf in tfs
+        bqtls_tf = TargeneCore.filter_bqtls_by_tf(bqtls, tf)
+        transactors_tf = TargeneCore.filter_transactors_by_tf(transactors, tf)
+        TargeneCore.parameters_from_actors(
+        bqtls_tf, transactors_tf, data, variables, orders, outprefix, tf; 
```

Comment:
I think it will make the code simpler to make a new `outprefix` for each TF instead of passing a new argument what do you think?

Reply:
outprefix has been updated to include the TF name: e4c2e131323f5032ff8a3be17776638ec0b1ba05 and e4c2e131323f5032ff8a3be17776638ec0b1ba05